docs/modules/ROOT/pages/documentation/microprofile/metrics/vendor-metrics.adoc

```diff
@@ -80,6 +80,25 @@ vendor_jdbc_connection_pool_numconnfree_total{name="resources/H2Pool"} 0
 vendor_jdbc_connection_pool_numconnfree_total{name="resources/__TimerPool"} 0 0
 ----
 
+==== Instance Name
+_Since Payara Server 5.2021.8_
+
+For the purpose of attributes specific to the running instance, Payara provides placeholder `${instance}`. It is replaced with the name of the current instance:
```

Comment:
```suggestion
Payara provides the placeholder `${instance}`. It is replaced with the name of the current instance:
```

Reply:
What do you mean by current instance? What happens if there are multiple instances?

Reply:
Applications and monitoring run on one instance in cluster. If it is DAS, the instance is "server". The monitoring doesn't combine metrics from multiple instances, so there is always only 1.